joomla/joomla-cms
# [4.0] Namespace com_admin

Pull Request for Issue # .

### Summary of Changes
As title says, this PR converts com_admin component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev branch
2. Apply patch
3. Access to Edit account link at top right corner, update admin account
4. Access to System - System Information
5. Access to Help - Joomla Help

Make sure all the features in 3, 4, 5 works
